### ISSUE TITLE
Инф-ция по комбинату

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#2311]
Originally reported on Google Code with ID 2311

```
Есть форма Информация по комбинату.
строка "Отгружено по отвесам с эксп.:" в ней пробивается кол-во отвесов и вес общий
по экспедициям (т.е. и по холодильнику тоже).
По кнопке формируется отчет - в нем вес правильный - по экспедиции только.
Нужно чтобы в форме был вес как и в отчете, т.е. только по подразделению экспедиция,

также в форму добавить строку "Отгружено по отвесам с хол-ка:"
и по кнопке отчет, как по экспедиции, только по подразделению холодильник.

Укажите дополнительную информацию ниже:

желательно до 3-4 февраля

```

Reported by `kmk_igor%tut.by@gtempaccount.com` on 2011-01-27 08:17:09

**Comment by gsbelarus** [comment]
Reported by `flakekun` on 2011-01-31 09:42:21
- **Status changed**: `Duplicate`
- **Merged into**: #2312

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2011-02-11 15:26:37
- **Labels added**: KMK
